videointelligence/google/cloud/videointelligence_v1/gapic/transports/video_intelligence_service_grpc_transport.py

```diff
@@ -19,6 +19,7 @@
 import google.api_core.operations_v1
 
 from google.cloud.videointelligence_v1.proto import video_intelligence_pb2_grpc
+from google.longrunning import operations_pb2
```

Comment:
This module is generated by the `gapic-generator`:  we cannot make manual changes to it.

Reply:
Removed.